src/types/onyx/Report.ts

```diff
@@ -170,6 +175,9 @@ type Report = OnyxCommon.OnyxValueWithOfflineFeedback<
         isLoadingPrivateNotes?: boolean;
         selected?: boolean;
 
+        /** Pending member of the report */
+        pendingVisibleChatMembers?: PendingChatMember[];
```

Comment:
Why `pendingVisibleChatMembers` and not just `pendingChatMembers`? Do we have `pendingInvisibleChatMembers`? 

(same comment everywhere this naming is used in this PR)